src/main/java/com/epam/tc/hw9/service/YandexSpellerAssertions.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.tc.hw9.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.hw9.dto.YandexSpellerDto;
+
+public class YandexSpellerAssertions {
+
+    public void checkTheCorrectWord(String text, String expected) {
+        YandexSpellerDto[] correctWord = new YandexSpellerService().getCorrectWord(text);
+        assertThat(correctWord[0].getS()).as("Parameter of right words not contains expected correct one")
```

Comment:
"Parameter of right words not contains expected correct one" - what does that mean?